src/chocolatey/infrastructure.app/commands/ChocolateyOutdatedCommand.cs

```diff
@@ -52,6 +52,9 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                 .Add("cp=|certpassword=",
                      "Certificate Password - the client certificate's password to the source. Defaults to empty. Available in 0.9.10+.",
                      option => configuration.SourceCommand.CertificatePassword = option.remove_surrounding_quotes())
+                .Add("pre|prerelease",
+                    "Prerelease - Include Prereleases? Defaults to false. Available in 0.10.14+.",
+                    option => configuration.Prerelease = option != null)

```

Comment:
It's amazing how simple of a change this was. Love it when things work out nicely like this!

Reply:
When code is written correctly, things can be easy!